client/src/components/Nav/NavProfile.jsx

```diff
@@ -17,18 +16,18 @@ class NavProfile extends Component {
       <React.Fragment>
         {
           isLogged() ? (
-            <NavItem>
-              <li><Link to={"/profile/" + sessionStorage['User-id']}>
-              <img
-                alt="user pictogram"
-                className="user-pictogram"
-                src={this.get_pictogram_url()}
-              />
-              &nbsp;
+            <li>
+              <Link to={"/profile/" + sessionStorage['User-id']}>
+                <img
+                  alt="user pictogram"
+                  className="user-pictogram"
+                  width="32"
```

Comment:
Imagine if it will be open on screen with large resolution, "width=32" will be enough?

Reply:
First, it's not part of current PR.
Second, maybe I skip something when I read materializecss source but the biggest size of nav is 64px.
Third, we don't have any validation or ready and working code for images now, so this property is temporary